client/components/DataManagement/DataManagementRow/index.jsx

```diff
@@ -553,7 +553,7 @@ const DataManagementRow = ({
                         <VersionString
                             role="listitem"
                             iconColor="#2BA51C"
-                            linkHref={`/test-review/${latestVersion.gitSha}/${latestVersion.testPlan.directory}`}
+                            linkHref={`/test-review/${latestVersion.id}`}
```

Comment:
It seems like the app isn't so heavily used or linked to that this will matter but just putting a note here to make sure that we all agree that breaking previous sha-based routing is okay.

Reply:
There is something nice about having the `testPlan.directory`in the URL. It feels more human-readable. Not requesting a change but more a point of discussion.

Reply:
I do feel like we still have enough leeway to change the URLs, I really doubt there are any bookmarks out there, but I appreciate you mentioning that!

For the URLs, my main motivation was to avoid using the whole git sha which made the URL feel unwieldy, but another option could be `/test-review/checkbox-two-state/v23.08.23`. This would require adding graphql support to query by versionString so it's a bit more work, but if you like it better maybe it's worth it?

That said, we generally use the TestPlanVersion ids elsewhere so this URL is pretty in keeping with our other URLs.

Reply:
That would definitely be the most human-readable URL but even just using the id with the test plan directory would add some info without adding a new query, like `/test-review/checkbox-two-state/5`. I tend to be biased towards putting as much human-readable information in the URL as possible but I definitely do not feel strongly about this. All of the options, including keeping as-is, work for me.